packages/governance-sdk/src/tools/borsh.ts

```diff
@@ -102,3 +102,9 @@ export function deserializeBorsh(
   const reader = new BinaryReader(buffer);
   return deserializeStruct(schema, classType, reader);
 }
+
+export abstract class BorshClass<T> {
+  constructor(args: T) {
+    Object.assign(this, args);
+  }
+}
```

Comment:
`BorshClass` is just a simple helper class that assigns the keys from the args object to properties on the instance

Reply:
this could probably be a decorator tbh but I didn't want to get _too_ weird with it yet